bin/rb/src/main.rs

```diff
@@ -37,22 +38,26 @@ impl Cli {
 enum Commands {
     Erc20(Erc20Args),
     Pool(PoolArgs),
+    Chain(ChainArgs),
 }
 
 #[tokio::main]
 async fn main() -> eyre::Result<()> {
     let cli = Cli::parse();
     match cli.into_command() {
         (Commands::Erc20(args), provider) => match args.command {
-            Some(Erc20Commands::Symbol(args)) => erc20_symbol(args, provider).await,
-            None => unreachable!(),
+            Erc20Commands::Symbol(args) => erc20_symbol(args, provider).await,
         },
         (Commands::Pool(args), _) => match args.command {
-            Some(PoolCommands::TickSpacing) => pool_tick_spacing().await,
-            Some(PoolCommands::CurrentTick) => pool_current_tick_command().await,
-            Some(PoolCommands::Dump) => pool_tick_dump_command().await,
-            Some(PoolCommands::Info) => pool_tick_info_command().await,
-            None => unreachable!(),
+            PoolCommands::TickSpacing => pool_tick_spacing().await,
+            PoolCommands::CurrentTick => pool_current_tick_command().await,
+            PoolCommands::Dump => pool_tick_dump_command().await,
+            PoolCommands::Info => pool_tick_info_command().await,
+        },
+        (Commands::Chain(args), provider) => match args.command {
+            commands::ChainCommands::ID => chain_id(provider).await,
```

Comment:
Some command fns have _command suffix and some don't. Prefer without suffix but happy either way as long as all fns do the same